templates/website/.eslintrc.cjs

```diff
@@ -29,6 +29,13 @@ module.exports = {
         'perfectionist/sort-vue-attributes': 'off',
       },
     },
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        'perfectionist/sort-imports': 'warn',
```

Comment:
Why set this to warn? This is auto-fixable and should be fine as errors

Reply:
we swap these out programmatically in payload config for templates when you choose a db adapter